app/facades/revenue_facade.rb

```diff
@@ -0,0 +1,7 @@
+class RevenueFacade
+  def self.merchant_revenue(merchant)
```

Comment:
Learning more and more about refactoring. Having the facade handle the AR query makes alot of sense